health.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 	"bufio"
 	"os"
+	"github.com/kkdai/maglev"
```

Comment:
It seems that the person who wrote this Maglev implementation did so as part of [a 52 week learning project](https://github.com/kkdai/project52). The Maglev implementation doesn't have a license, but the larger project is [licensed under the MIT license](https://github.com/kkdai/project52/blob/master/LICENSE). Perhaps we should ping the author to be safe.